packages/api-headless-cms/src/plugins/graphql/install.ts

```diff
@@ -0,0 +1,24 @@
+import { install, isInstalled } from "./installResolver/install";
+
+export default {
+    typeDefs: /* GraphQL */ `
+        type CmsBooleanResponse {
```

Comment:
Please put this into the main graphql file, which imports all of these smaller ones. Just so it's more clear that `CmsBooleanResponse` is there, and other files can rely on it.